centreon/www/front_src/src/Resources/Listing/columns/Status.tsx

```diff
@@ -166,14 +166,18 @@ const StatusColumn = ({
 
     const statusName = row.status.name;
 
+    const label = equals(SeverityCode[5], statusName)
```

Comment:
```suggestion
    const label = equals(SeverityCode.OK, statusName)
```

Reply:
to make the comparison with the name  of status ok , this way it returns the code of status 5 👀